hyperspy/models/eelsmodel.py

```diff
@@ -887,6 +887,8 @@ def free_edges(self, edges_list=None):
         for edge in edges_list:
             if edge.isbackground is False:
                 edge.intensity.free = True
```

Comment:
Use `set_parameters_free` instead?

Reply:
This would set the `effective_angle` parameter also to free, which I don't think would be the intention.

Reply:
Not if you specify the parameter list: https://hyperspy.readthedocs.io/en/latest/api/hyperspy.component.html#hyperspy.component.Component.set_parameters_free